module/preference/PreferenceDialog.kt

```diff
@@ -91,18 +93,18 @@ object PreferenceDialog {
 
     private val preferenceMgr = StudioState.preference
 
-    private var focusedPreferenceGroup by mutableStateOf<PreferenceGroup>(PreferenceGroup.Root(emptyList()))
+    private var selectedPreferenceGroup by mutableStateOf<PreferenceGroup>(PreferenceGroup.Root())
     private var state by mutableStateOf(PreferencesForm())
 
-    sealed class PreferenceField(private val label: String, private val caption: String?) {
+    sealed class PreferenceField(private val label: String, private val caption: String?, private val fieldHeight: Dp = Form.FIELD_HEIGHT) {
         abstract fun isValid(): Boolean
         @Composable abstract fun Display()
 
         var modified by mutableStateOf(false)
 
         @Composable
         fun Layout(fieldContent: @Composable () -> Unit) {
-            Field(label) {
+            Field(label, fieldHeight) {
```

Comment:
We're not assuming all fields are the same height anymore, and provide a `fieldHeight` that multiplies the height of the field.